packages/graphql/mock-server/README.md

```diff
@@ -0,0 +1,151 @@
+# GraphQL Mock Server
+
+> Faster React component development using GraphQL Mock-Server (Apollo)
+
+When you are using GraphQL an client side to fetch and bind data into your UI components, it's quite often necessary to work with mock/stub data. There exists tons of feasible reasons why mocking makes sense in daily practices. In summary, the following seem to be at least important.
+
+- GraphQL schema design in a Frontend-Driven approach
+- Switching between local and remote query execution to work autonomously without an online GraphQL-Server access
+- Faster execution of component integration test using local mock data sets
+- Mock data set support to prove experimental/feature functionality thesis
+
+## Setup
+
+**Install hops-graphql preset**
+
+`yarn add hops-graphql --dev`
+
+**Start Hops development**
+
+`hops start`
+
+**Supports Local GraphQL Playground against your GraphQL schema**
+
+`open http://localhost:<port>/graphql`
+
+![GraphiQL Playground](./playground.png)
+
+## Mock-Server Configuration
+
+There are two combinable ways to configure the mock server.
+
+- `remoteSchemas` - a list of remote GraphQL servers for which this mock server assumes a proxy role. When the mock server starts, the schema is downloaded from the url and exported as well. Additionally, you have the possibility to overwrite parts of the proxy schema with your own mock dataset resolvers.
+- `localMocks` - a list of local GraphQL schemas to define your own GraphQL schemas and resolvers or extend an exisiting local or remote schema.
+
+```json
+{
+  "hops": {
+    "remoteSchemas": [
+      {
+        "url": "https://api.github.com/graphql",
+        "headers": {
+          "Authorization": "bearer [GITHUB_API_TOKEN]"
```

Comment:
ok

Reply:
@ZauberNerd @dmbch https://github.com/xing/hops/pull/590/commits/5f1ae039e59e0e1d35e87158234c376406c3792d

Reply:
We should not have this as an example in here, because this will expose secret environment variables.
Maybe you can change it to use an external graphql mock config file?